src/components/Button.js

```diff
@@ -12,8 +13,7 @@ export const VARIANTS = [
   'image',
 ];
 
-
-const Button = ({ type='', component: Component='button', variant='', onDisabledClick, children, ...extra }) => {
+const Button = ({ type='', component: Component=UtrechtButton, variant='', onDisabledClick, children, ...extra }) => {
```

Comment:
The proptype for `component` needs to be updated too - it now expects `string` only, but we're actually passing a component. Storybook warning:

```
Warning: Failed prop type: Invalid prop `component` of type `function` supplied to `Button`, expected `string`.
```